test/application/widgets/theme/custom_text_field_test.dart

```diff
@@ -12,7 +12,7 @@ void main() {
       final textField = CustomTextField(controller: controller);
```

Comment:
is this `pumpThemedProviderScoped` even used?

Reply:
It was renamed to this new `pumpProviderScoped`. I forgot to rebase some of the PRs and this is currently breaking `main` branch. I added the fix within this PR